ic-os/components/networking/nftables/nftables-add-operator-rules-guestos.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=/usr/bin/bash -c 'set -o pipefail ; /opt/ic/bin/guestos_tool render-firewall-config | nft -f -'
```

Comment:
Can you please test: if this service fails, will the nftables configuration fail completely (i.e. node becomes wide open), or is only the node provider configuration going to be missing (i.e. node provider can't scrape logs/metrics)?

Reply:
Yes this was already tested, if the service fails (because the second command I added fails), then the service appears failed, but the rules from `/etc/nftables.conf` get applied anyway.

I triggered the failure intentionally in both HostOS and GuestOS.